curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-046.md

```diff
@@ -1,34 +1,80 @@
+Rather than having to constantly double-check you are not overwriting your earlier properties, you can use the `!important` keyword to ensure these properties are always applied, regardless of order or specificity.
 
-Create a `.row[tabindex]:focus` selector, and give it a `background-color` property set to `#198eee` and a `color` property set to `white`. Then try experimenting with hovering and clicking on your rows.
+Give each property in your `span[class~="sr-only"]` selector an `!important` keyword. Do not change any of the values.
 
 # --hints--
 
-You should have a new `.row[tabindex]:focus` selector.
+Your `span[class~="sr-only"]` selector should have the `border` property set to `0 !important`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row[tabindex]:focus'));
+// log it
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('border: 0px !important;') || text.includes('border: 0px none !important'));
 ```
 
-Your `.row[tabindex]:focus` selector should have a `background-color` property set to `#198eee`.
+Your `span[class~="sr-only"]` selector should have the `clip` property set to `rect(1px, 1px, 1px, 1px) !important`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row[tabindex]:focus')?.backgroundColor === 'rgb(25, 142, 238)');
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('clip: rect(1px, 1px, 1px, 1px) !important;'));
 ```
 
-Your `.row[tabindex]:focus` selector should have a `color` property set to `white`.
+Your `span[class~="sr-only"]` selector should have the `clip-path` property set to `inset(50%) !important`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row[tabindex]:focus')?.color === 'white');
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('clip-path: inset(50%) !important;'));
+```
+
+Your `span[class~="sr-only"]` selector should have the `-webkit-clip-path` property set to `inset(50%) !important`.
+
+```js
+// this one gets removed apparently
+assert(code.includes('-webkit-clip-path: inset(50%) !important;'));
+```
+
+Your `span[class~="sr-only"]` selector should have the `height` property set to `1px !important`.
+
+```js
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('height: 1px !important;'));
+```
+
+Your `span[class~="sr-only"]` selector should have the `width` property set to `1px !important`.
+
+```js
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('width: 1px !important;'));
+```
+
+Your `span[class~="sr-only"]` selector should have the `position` property set to `absolute !important`.
+
+```js
+const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
+assert(text.includes('position: absolute !important;'));
+```
+
```

Comment:
Any reason why these tests are not there?

```suggestion
Your `span[class~="sr-only"]` selector should have the `overflow` property set to `hidden !important`.

const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
assert(text.includes('overflow: hidden !important;'));

Your `span[class~="sr-only"]` selector should have the `white-space` property set to `nowrap !important`.

const text = new __helpers.CSSHelp(document).getStyle('span[class~="sr-only"]')?.cssText;
assert(text.includes('white-space: nowrap !important;'));
```

Reply:
Because at some point I lost the properties and had to put them back